cache-config/t3c-apply/config/config.go

```diff
@@ -329,25 +328,25 @@ func GetCfg() (Cfg, error) {
 	var tsHome = ""
 	if *tsHomePtr != "" {
 		tsHome = *tsHomePtr
-		fmt.Printf("set TSHome from command line: '%s'\n\n", TSHome)
+		log.Infof("set TSHome from command line: '%s'\n\n", TSHome)
```

Comment:
Everything with multiple newlines needs changed to only have 1. E.g. this needs changed to 

`log.Infof("set TSHome from command line: '%s'\n", TSHome)`

Multiple newlines will result in log lines without the required prefix, which breaks people's log parsers.

I made a PR to make that impossible by always prefixing every newline in the log function. But people didn't like that.

Reply:
Changed to single new line.